pages/add.js

```diff
@@ -0,0 +1,125 @@
+import Head from 'next/head'
+
+import { useState } from 'react';
+import { supabase } from '../utilities/supabase';
+
+import { useUser } from '@supabase/auth-helpers-react'
+
+import { Modal } from '../components/container/modal';
+
+import { useRouter } from 'next/router';
+
+import styles from '../styles/container/add.module.scss';
+
+function HomePage() {
+
+    const user = useUser();
+    const router = useRouter();
+
+    let newDate = new Date()
+    let date = newDate.getDate();
+    let month = newDate.getMonth() + 1;
+    let year = newDate.getFullYear();
+
+    const months = ["Ianuarie", "Februarie", "Martie", "Aprilie", "Mai", "Iunie", "Iulie", "August", "Septembrie", "Octombrie", "Noiembrie", "Decembrie"]
+    
+    const theMonth = months[month - 1]
+
+    const theDate = `${date} ${theMonth} ${year}`
+
+    const [expression, setExpression] = useState();
+    const [explication, setExplication] = useState();
+    const [example, SetExample] = useState();
```

Comment:
Numele funcțiilor simple ar trebui să înceapă cu literă mică

```suggestion
    const [example, setExample] = useState();
```